feature(tour) add tour option to homepage

jomadoye opened the pull request:
#### What does this PR do?
  Adds a tour option to the home 

#### Description of Task to be completed?
As a new user logged into the application, I should see an Introductory walkthrough of how to use Reaction Commerce.

#### How should this be manually tested?
1) Sign into the application
2) Click on the tour button
3) End the tour

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#144187573 Getting Started Tour

jomadoye committed:
feature(tour) add tour option to homepage

[144187575]

jomadoye committed:
feature(tour) style tour to match updated style format

[144187575]

tonymontaro commented on .meteor/versions:
```diff
@@ -1,10 +1,10 @@
 accounts-google@1.1.2
 accounts-oauth@1.1.15
-accounts-password@1.3.5
+accounts-password@1.3.6
```
Please revert these version numbers back if they did not affect what you did, changing them usually break the Heroku deployment.

jomadoye replied:
@andela-angene I need them for my feature :-), please fix this your wayo wayo heroku deployment.

tonymontaro commented on imports/plugins/included/tour/client/tour.js:
```diff
@@ -0,0 +1,387 @@
+const tour = introJs.introJs();
+const adminTourSteps = [
+  {
+    intro: `<h2>Welcome to Reaction Commerce</h2>
```
Change this to Kakashi Reaction Commerce

jomadoye replied:
would do this now.

tonymontaro reviewed (approved):
Good Job!

jomadoye committed:
feature(tour) update the tour guide

[144187575]

jomadoye committed:
feature(tour) update version numbers

This is necessary, so as not to break the wayo wayo heroku deployment, tony did :-).
